docs/conceptual/system-speed-of-light.rst

```diff
@@ -0,0 +1,318 @@
+.. meta::
+   :description: Omniperf performance model: System Speed-of-Light
+   :keywords: Omniperf, ROCm, profiler, tool, Instinct, accelerator, AMD, system, speed of light
+
+*********************
+System Speed-of-Light
+*********************
+
+System Speed-of-Light summarizes some of the key metrics from various sections
+of Omniperf’s profiling report.
+
+.. warning::
+
+   The theoretical maximum throughput for some metrics in this section are
+   currently computed with the maximum achievable clock frequency, as reported
+   by :doc:`rocminfo:index`, for an accelerator. This may not be realistic for
```

Comment:
it isn't reported by the rocminfo documentation, but by the rocminfo _tool_.  I would phrase this as "as reported by ...`rocminfo`, the documentation for which can be found ..."

Reply:
fixed